examples/zephyr/fw_update/pytest/conftest.py

```diff
@@ -17,13 +18,15 @@ async def fw_info():
 
 @pytest.fixture(scope="module")
 async def blueprint_id(project, request):
-    bp_name = request.config.option.platform.replace('/','_')
+    bp_name = request.config.option.platform[0].replace("/", "_")
```

Comment:
This is actually a different `--platform` than the one we pass to twister. The `--platform` here is from the value that twister passes to pytest.